Kinvey/Kinvey/Date.swift

```diff
@@ -9,9 +9,17 @@
 import Foundation
 
 extension Date {
+    
+    static let dateWriteFormatter: DateFormatter = {
+        let wFormatter = DateFormatter()
+        wFormatter.locale = Locale(identifier: "en_US_POSIX")
+        wFormatter.timeZone = TimeZone(identifier: "UTC")
+        wFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+        return wFormatter
+    }()
 
     public func toISO8601() -> String {
-        return KinveyDateTransform().transformToJSON(self)!
+        return Date.dateWriteFormatter.string(from: self)
```

Comment:
@heyzooi - Because we are changing the format of what we are returning here and this is a `public func`, do we have to worry about this being a breaking change?

Reply:
the format output is the same, I just made some changes here for performance reasons, so instead create a new instance and call a method every time, I'm using the static variable so we will have a better performance